assets/src/scripts/components/graph/state/well-log.js

```diff
@@ -179,6 +181,49 @@ const getRadiusScale = memoize((opts, chartType) => createSelector(
     }
 ));
 
+const createBoreholeElement = function(opts, xScale, yScale, elements, wellDepth) {
+    // scale the depth to the graph size
+    const scaleDepth  = yScale(wellDepth);
+    // check to see if we have construction elements
+    const hasElements = elements && elements.length > 0;
+    // if there are no elements then use max size of the graph, else use 0 for max function to work
+    const minDepth    = hasElements ? 520 : 0;
```

Comment:
Where is the max size of the graph defined (where did 520 and 300 come from)? I couldn't find them. Should you be using the size of the container (getContainerSize selector)?

Reply:
I am not sure how the graph was defined. I will investigate where those values come from. Yes, of course I typed the in, but I mean how they came about in the first place. They are not mine. They are the size of the plot.